src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -929,4 +934,195 @@ public void testReverseDepositThatRepaysOverdraft() throws SQLException, ScriptE
     assertEquals(CUSTOMER1_OVERDRAFT_BALANCE_IN_PENNIES, (int)customer1Data.get("OverdraftBalance"));
   }
 
+ /**
+   * This test verifies that a simple transfer of $100 from Customer1 to Customer2 will take place. Customer1's balance will be
+   * initialized to $1000, and Customer2's balance will be $500. 
+   * 
+   * After a successful transfer, Customer1's balance should reflect a $900 balance, and Customer2's balance should be $600. 
+   * 
+   * @throws SQLException
+   */
+  @Test
+  public void testTransfer() throws SQLException, ScriptException { 
+
+    //Initialize customer1 with a balance of $1000. Balance will be represented as pennies in DB.
+    double CUSTOMER1_BALANCE = 1000;
+    int CUSTOMER1_BALANCE_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER1_BALANCE);
+    MvcControllerIntegTestHelpers.addCustomerToDB(dbDelegate, CUSTOMER1_ID, CUSTOMER1_PASSWORD, CUSTOMER1_FIRST_NAME, CUSTOMER1_LAST_NAME, CUSTOMER1_BALANCE_IN_PENNIES);
+
+    //Initialize customer2 with a balance of $500. Balance will be represented as pennies in DB. 
+    double CUSTOMER2_BALANCE = 500;
+    int CUSTOMER2_BALANCE_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER2_BALANCE);
+    MvcControllerIntegTestHelpers.addCustomerToDB(dbDelegate, CUSTOMER2_ID, CUSTOMER2_PASSWORD, CUSTOMER2_FIRST_NAME, CUSTOMER2_LAST_NAME, CUSTOMER2_BALANCE_IN_PENNIES);
+
+    //Amount to transfer
+    int TRANSFER_AMOUNT = 100;
```

Comment:
Make this a `double` to follow the conventions of the application. User input should always be considered a `double`, even if it is a whole-dollar amount like $100.